EverythingToolbar/Controls/FilterSelector.xaml.cs

```diff
@@ -47,6 +63,11 @@ private void OnComboBoxItemSelected(object sender, SelectionChangedEventArgs e)
             if (ComboBox.SelectedIndex < 0)
                 return;
 
+            if (!ComboBox.IsFocused && !ComboBox.IsMouseOver) { 
```

Comment:
During testing this code was reached (and toggles selection) when an item is selected from the dropdown list, there wasn't another reliable property I could find that would indicate that the window is actually open (during the `Show()` triggering of this event, even though the window wasn't yet visible, both `isVisible` and `isHitBoxVisible` were `true`.